sql-plugin/src/main/scala/org/apache/spark/sql/rapids/test/cpuJsonExpressions.scala

```diff
@@ -0,0 +1,419 @@
+/*
+ * Copyright (c) 2024, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql.rapids.test
+
+import java.io.{ByteArrayOutputStream, StringWriter}
+import java.util.Objects
+
+import scala.collection.mutable.ArrayBuffer
+import scala.util.parsing.combinator.RegexParsers
+
+import ai.rapids.cudf.{ColumnVector, DType, HostColumnVector, Table}
+import com.fasterxml.jackson.core.{JsonEncoding, JsonFactoryBuilder, JsonGenerator, JsonParser, JsonProcessingException, JsonToken}
+import com.fasterxml.jackson.core.json.JsonReadFeature
+import com.nvidia.spark.rapids.{DumpUtils, GpuColumnVector, GpuScalar}
+import com.nvidia.spark.rapids.Arm.withResource
+
+import org.apache.spark.sql.catalyst.json.CreateJacksonParser
+import org.apache.spark.unsafe.types.UTF8String
+import org.apache.spark.util.Utils
+
+object CpuGetJsonObject {
+
+  private val maxRowsToSave = 1024 * 10
```

Comment:
Let's make this configurable as an internal configuration.

Reply:
Done.